src/proto/Cosmos.proto

```diff
@@ -88,9 +95,10 @@ message SigningInput {
         StakeMessage unstake_message = 9;
         ReStakeMessage restake_message = 10;
         WithdrawStakeRewardMessage withdraw_stake_reward_message = 11;
+        WithdrawStakeRewardsAllMessage withdraw_stake_rewards_all_message = 12;
     }
     // default is cosmos-sdk/MsgSend
-    string type_prefix = 12;
+    string type_prefix = 20;
```

Comment:
I don't understand this change, it's probably right; please check

Reply:
we added a new message type, so index also needs to be increased, 13~19 are reserved for new msgs